tests/e2e/core-tests/orders_03_multiple_vendor_test.js

```diff
@@ -1,7 +1,7 @@
 const helpers = require('../pages/helpers');
 Feature('Multiple order Multiple vendor Functionality');
-Scenario('test something 1', ({ I }) => {
-    I.loginAsCustomer();
+Scenario('test something 1', ({ I,loginAs}) => {
+    loginAs('customer');
```

Comment:
Maintain upper and lower case for the user role because the user is case sensitive.